BankSDK/GiniBankSDK/Tests/GiniBankSDKTests/TransactionDocsDocumentPagesViewModelTests.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  TransactionDocsDocumentPagesViewModelTests.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import XCTest
+@testable import GiniBankSDK
+
+final class TransactionDocsDocumentPagesViewModelTests: XCTestCase {
+    private let testIban = "IBAN12345678901234567890"
+    private let validAmountToPay = Price(value: 100, currencyCode: "EUR")
+    private let zeroAmountToPay = Price(value: 0, currencyCode: "EUR")
+    private let testImages = [UIImage(), UIImage()]
+    private let singleTestImage = [UIImage()]
+
+    private func buildViewModel(amount: Price, iban: String, images: [UIImage]) -> TransactionDocsDocumentPagesViewModel {
+        let extractions = TransactionDocsExtractions(amountToPay: amount, iban: iban)
+        return TransactionDocsDocumentPagesViewModel(originalImages: images, extractions: extractions)
+    }
+
+    func testInitializationWithValidAmountToPayAndIBAN() {
+        let viewModel = buildViewModel(amount: validAmountToPay, iban: testIban, images: testImages)
+
+        XCTAssertEqual(viewModel.bottomInfoItems.count, 2)
+        XCTAssertTrue(viewModel.bottomInfoItems.contains(viewModel.amountToPayString))
+        XCTAssertTrue(viewModel.bottomInfoItems.contains(viewModel.ibanString))
+    }
+
+    func testInitializationWithZeroAmountToPay() {
+        let viewModel = buildViewModel(amount: zeroAmountToPay, iban: testIban, images: singleTestImage)
+
+        XCTAssertEqual(viewModel.bottomInfoItems.count, 1)
+        XCTAssertFalse(viewModel.bottomInfoItems.contains(viewModel.amountToPayString))
+        XCTAssertTrue(viewModel.bottomInfoItems.contains(viewModel.ibanString))
+    }
+
+    func testInitializationWithEmptyIBAN() {
+        let viewModel = buildViewModel(amount: validAmountToPay, iban: "", images: singleTestImage)
+
+        XCTAssertEqual(viewModel.bottomInfoItems.count, 1)
+        XCTAssertTrue(viewModel.bottomInfoItems.contains(viewModel.amountToPayString))
+        XCTAssertFalse(viewModel.bottomInfoItems.contains(viewModel.ibanString))
```

Comment:
[suggestion] Can we please add an explanation in case of failure:
```XCTAssertEqual(viewModel.bottomInfoItems.count, 2, "Expected two bottom info items when both IBAN and amount are present")
XCTAssertTrue(viewModel.bottomInfoItems.contains(viewModel.amountToPayString), "Expected amount to be present in bottom info items")
XCTAssertFalse(viewModel.bottomInfoItems.contains(viewModel.amountToPayString), "Expected amount not to be present when zero")
````

Please do this for every test. Thank you!